INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql

```diff
@@ -464,7 +466,11 @@ END AS tradingNames,
 		ISNULL((SELECT phone from ci_office where office_name LIKE 'DRO%'), '')
 		ELSE 
 		(insolvencyService.phone)
-	END AS insolvencyServicePhone
+	END AS insolvencyServicePhone,
+	CASE WHEN 
+		discharge.previous_order_status IN ('D', '', NULL) AND discharge.previous_order_date IS NOT NULL AND discharge.previous_order_date BETWEEN DATEADD(yy, -6, inscase.insolvency_date) AND inscase.insolvency_date 
+	then 
+		discharge.previous_order_status else null end as dateOfPreviousOrder
```

Comment:
can you update the keywords to upper case  then, else, null, end, as, to keep the readability and consistency of the proc